index.html

```diff
@@ -31,7 +31,139 @@ <h1>I’m Mariane
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
+     <button type="button">See Project</button>
+    </article>
+   </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot1.png" alt=""></article>
+    <article>
+     <h1>Multi-Post Stories</h1>
+     <div class="frame">
+      <h2>CANOPY</h2>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
+     <button type="button">See Project</button>
+    </article>
+   </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot2.png" alt=""></article>
+    <article>
+     <h1>Tonic</h1>
+     <div class="frame">
+      <h2>CANOPY</h2>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
+     <button type="button">See Project</button>
+    </article>
+   </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot3.png" alt=""></article>
+    <article>
+     <h1>Multi-Post Stories</h1>
+     <div class="frame">
+      <h2 class="tc">CANOPY</h2>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
+     <button type="button">See Project</button>
+    </article>
+   </section>
   </main>
-  <footer></footer>
+  <footer>
+   <section class="Fabout">
+    <h1>About<br>Myself</h1>
+    <p>Hello I’m a software developer! 
+      I can help you build a product , 
+      feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.</p>
+    <a href="">LET’S CONNECT</a>
+    <ul class="social_media">
+     <li><a href=""><img src="images/vector1.svg" alt=""></a></li>
+     <li><a href=""><img src="images/link.svg" alt=""></a></li>
+     <li><a href=""><img src="images/vector2.svg" alt=""></a></li>
+     <li><a href=""><img src="images/twiter.svg" alt=""></a></li>
+     <li><a href=""><img src="images/vector.svg" alt=""></a></li>
```

Comment:
- [ ] [OPTIONAL]: You should consider using icons instead of images in this case. Icons are more customizable, you can easily change colors and sizes. I recommend using [font awesome](https://fontawesome.com/), but there are plenty free icon libraries online.